.github/workflows/labeler.yaml

```diff
@@ -13,5 +13,5 @@ jobs:
     steps:
     - uses: actions/labeler@v5
       with:
-        repo-token: "${{ github.token }}"
+        repo-token: "${{ secrets.RHACS_BOT_GITHUB_TOKEN }}"
```

Comment:
Is it necessary to use the custom token here? Looking at [the documentation](https://github.com/actions/labeler?tab=readme-ov-file#notes-regarding-pull_request_target-event), it might be enough to change the trigger to `pull_request_target` (which you did) and potentially setting increased permissions for the default token.